tests/phpunit/admin/class-wc-rest-stripe-agentic-commerce-controller-test.php

```
@@ -0,0 +1,336 @@
+						'code'    => 200,
+						'message' => 'OK',
+					],
+					'headers'  => [],
+					'body'     => wp_json_encode( [ 'id' => 'file_stub' ] ),
+				];
+			},
+			10,
+			3
+		);
+
+		$request  = new WP_REST_Request( 'POST', self::REST_BASE . '/sync' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '303', column '19'). (undefined)

(MissingImport)